pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,70 @@
+{stdenv, python3Packages, gettext, qt5, fetchFromGitHub}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ 
+    gettext 
+    python3Packages.pyqt5
+    qt5.wrapQtAppsHook
+  ];
+
+  pythonPath = with python3Packages; [ 
+    pyqt5
+    send2trash
+    sphinx
+    polib
+    hsaudiotag3k  
+  ];
+
+  format = "other";
```

Comment:
This is typically placed between `pname`+`version` and `src` to make it immediately obvious.

Reply:
fixed